WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -3049,6 +3065,51 @@ public void onTrackableEvent(TrackableEvent event, Map<String, String> propertie
         mEditorTracker.trackEditorEvent(event, mEditorFragment.getEditorName(), properties);
     }
 
+    @Override public void onStoryComposerLoadRequested(ArrayList<Object> mediaFiles, String blockId) {
+        LoadStoryFromStoriesPrefsUseCase loadStoryFromStoriesPrefsUseCase = new LoadStoryFromStoriesPrefsUseCase(
+                new StoryRepositoryWrapper(),
+                mSite,
+                mMediaStore,
+                this
+        );
+        ArrayList<String> mediaIds =
+                loadStoryFromStoriesPrefsUseCase.getMediaIdsFromStoryBlockBridgeMediaFiles(mediaFiles);
+        boolean allStorySlidesAreEditable = loadStoryFromStoriesPrefsUseCase.areAllStorySlidesEditable(mSite, mediaIds);
+        boolean failedLoadingOrReCreatingStory = false;
+
+        // now look for a Story in the StoryRepository that has all these frames and, if not found, let's
+        // just build the Story object ourselves to keep these files arrangement
```

Comment:
When you say `build the Story object ourselves to keep these files arrangement` what do you mean by the `files arrangement` part?

Reply:
so basically is the newly re-created Story should match the same order in which the media files being passed came in in the first place. That's what I meant by files arrangement

Reply:
@mzorz understood. Thank you. I think it would be good to use this explanation in the comment instead of `files arrangement` as it's more clear. So something like 

```
 // now look for a Story in the StoryRepository that has all these frames and, if not found, let's
 // just build the Story object ourselves to match the order in which the media files were passed. 
```

Reply:
done in cadf22b4df